zip-0316.rst

```diff
@@ -678,15 +789,21 @@ The following derivations are applied to each component FVK:
 
 In each case, the Typecode remains the same as in the FVK.
 
-Items (including Metadata Items) that are unrecognized by a given
-Consumer, or that are specified in experiments that the user has not
-opted into (see `Experimental Usage`_), MUST be dropped when deriving
-a UIVK from a UFVK.
+Items (including Metadata Items that are not "MUST-understand") that
+are unrecognised by a given Consumer, or that are specified in experiments
+that the user has not opted into (see `Experimental Usage`_), MUST be
+dropped when deriving a UIVK from a UFVK.
 
 
 Deriving a Unified Address from a UIVK
 --------------------------------------
 
+As of `Revision 1`_, a Consumer of a UIVK MUST understand the meaning
+of all "MUST-understand" Metadata Item Typecodes present in the UIVK
+in order to perform Unified Address derivation. For Metadata Items
```

Comment:
Express this as a consequence of the general MUST-understand requirement rather than restating it as if it were a separate conformance requirement.